File: pytools/examples/java/client/protocol/AckSceneExit.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckSceneExit
{
	private int uid;


	public AckSceneExit(Packet packet)
	{
		uid = packet.readInt();
	}



	public long getUid()
	{
		return PacketUtil.readUInt(this.uid);
	}

}

File: pytools/examples/csharp/client/Protocol/AckSceneExit.cs
using System.Collections;
using System.Collections.Generic;


public class AckSceneExit
{
	private uint _uid;


	public AckSceneExit(Packet packet)
	{
		this._uid = packet.ReadUint();
	}


	public uint uid
	{
		get { return this._uid; }
		set { this._uid = value; }
	}

}
